src/components/SelectionList/Search/ReportListItem.tsx

```diff
@@ -176,9 +162,10 @@ function ReportListItem<TItem extends ListItem>({
                             <View style={StyleUtils.getSearchTableColumnStyles(CONST.SEARCH_TABLE_COLUMNS.TYPE)} />
                             <View style={StyleUtils.getSearchTableColumnStyles(CONST.SEARCH_TABLE_COLUMNS.ACTION)}>
                                 <ActionCell
-                                    showTooltip={showTooltip}
-                                    isLargeScreenWidth={isLargeScreenWidth}
-                                    onButtonPress={handleOnButtonPress}
+                                    action={reportItem.action ?? 'view'}
+                                    transactionIDs={[]}
```

Comment:
Can we set a default value for transactionIDs in ActionCell and not pass an empty array here?